CHANGELOG.md

```diff
@@ -1,3 +1,141 @@
+# v1.10.0 (2022-10-10)
+
+## OS Changes
+* Automated reconciliation of boot settings ([#2375])
+* Support for static IP addressing ([#2204], [#2330], [#2445])
+* Add sources for NVIDIA's 515 driver ([#2455])
+* Set mode for tmpfs mounts ([#2473])
+* Increase inotify default limits ([#2335])
+* Increase `vm.max_map_count` to match AL2's configurations ([#2344])
+* Add support to configure DNS settings through the API ([#2353])
```

Comment:
This might read better as something like:
"Add support for configuring DNS settings..."